modules/seqexec/server/src/main/scala/seqexec/server/SeqTranslate.scala

```diff
@@ -353,17 +353,11 @@ class SeqTranslate(site: Site, systems: Systems[IO], settings: TranslateSettings
     sys: InstrumentSystem[IO]
   ): TrySeq[List[System[IO]]] = {
     stepType match {
-      case StepType.CelestialObject(inst) => for {
-        tcs  <- getTcs(inst.hasOI.fold(allButGaos, allButGaosNorOi), useGaos = false, sys,
-          TcsController.LightSource.Sky, config)
-        gcal <- Gcal.fromConfig(systems.gcal, site == Site.GS)(config)
-      } yield List(sys, tcs, gcal)
+      case StepType.CelestialObject(inst) => getTcs(inst.hasOI.fold(allButGaos, allButGaosNorOi), useGaos = false, sys,
+          TcsController.LightSource.Sky, config).map{ List(sys, _, Gcal.defaultGcal(systems.gcal)) }
 
-      case StepType.NodAndShuffle(inst)   => for {
-        tcs  <- getTcs(inst.hasOI.fold(allButGaos, allButGaosNorOi), useGaos = false, sys,
-          TcsController.LightSource.Sky, config)
-        gcal <- Gcal.fromConfig(systems.gcal, site == Site.GS)(config)
-      } yield List(sys, tcs, gcal)
+      case StepType.NodAndShuffle(inst)   => getTcs(inst.hasOI.fold(allButGaos, allButGaosNorOi), useGaos = false, sys,
+          TcsController.LightSource.Sky, config).map{ List(sys, _, Gcal.defaultGcal(systems.gcal)) }
```

Comment:
Why don't you need `site` anymore?

Reply:
There is a difference in when the IR lamp is turned on at both sites, but it does not matter when GCAL is not used.